src/python/pants/backend/shell/target_types.py

```diff
@@ -247,22 +247,28 @@ class ShellCommand(Target):
         ShellCommandSources,
         ShellCommandToolsField,
     )
-    help = dedent(
-        """\
-        Execute any external tool for its side effects.
-        This may be retried and/or cancelled, so ensure that it is idempotent.
-
-        Example BUILD file:
-
-            experimental_shell_command(
-              command="./my-script.sh --flag",
-              tools=["tar", "curl", "cat", "bash", "env"],
-              dependencies=[":scripts"],
-              outputs=["results/", "logs/my-script.log"],
-            )
+    help = (
+        "Execute any external tool for its side effects.\n"
+        + dedent(
```

Comment:
Yay thanks for doing that formatting :) This is great because Pants wraps lines based on your terminal width, but we need to use implicit string concatenation for that to work properly.

(The docs also work better when using implicit string concatenation so that the browser can wrap for you.)

Reply:
Yes, I remember you told me this once before ;)